deposit/src/test/java/edu/unc/lib/deposit/normalize/AbstractNormalizationJobTest.java

```diff
@@ -36,25 +35,25 @@
 import org.junit.Rule;
 import org.junit.rules.TemporaryFolder;
 import org.mockito.Mock;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
 
 import com.hp.hpl.jena.rdf.model.Model;
-import com.hp.hpl.jena.rdf.model.Property;
 import com.hp.hpl.jena.rdf.model.Resource;
-import com.hp.hpl.jena.rdf.model.StmtIterator;
-import com.hp.hpl.jena.vocabulary.RDF;
-
 import edu.unc.lib.deposit.AbstractDepositJobTest;
 import edu.unc.lib.deposit.work.AbstractDepositJob;
+import edu.unc.lib.dl.fcrepo4.PIDs;
+import edu.unc.lib.dl.fedora.PID;
 import edu.unc.lib.dl.rdf.Cdr;
 import edu.unc.lib.dl.rdf.CdrDeposit;
-import edu.unc.lib.dl.util.ContentModelHelper;
 import edu.unc.lib.dl.util.DepositStatusFactory;
 import edu.unc.lib.dl.util.JobStatusFactory;
 
 /**
  * @author bbpennel
  * @date Jun 18, 2014
  */
+@SuppressWarnings("deprecation")
```

Comment:
Yeah, unless we want to replace the deprecated XPath stuff at the bottom of the class. What do you think?